### PR TITLE
Update buy-a-domain-name.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note for tenants purchasing it through partner channel.

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/get-help-with-domains/buy-a-domain-name.md [hunk 1]
```diff
@@ -73,6 +73,9 @@ You can choose from the following top level domains for your domain.
     
 ![On the Manage domains page, click Buy domain](../media/16d6b3ba-ce13-496e-a7c0-1d99479dd203.png)
 
+> [!NOTE]
+> when you click on Buy domain, You may be redirected to your microsoft partner's website if the tenant is Purcahsed/Managed through a  Microsoft partner.
```
Comment (JohanFreelancer9) [inline Office365-Admin/get-help-with-domains/buy-a-domain-name.md]:
```suggestion
> When you click on **Buy domain**, you may be redirected to your Microsoft partner's website if the tenant is purchased/managed through a  Microsoft partner.
```

Comment (nenonix) [reply Office365-Admin/get-help-with-domains/buy-a-domain-name.md]: agree

**Comment by mypil** [comment]
@msbemba - please merge the commit on the copy/edit review made by JohanFreelancer9 / nenonix  or make the necessary changes on the PR following the suggestions made. Thank you.

**Pull request merged**